tox.ini

```diff
@@ -36,3 +36,6 @@ deps =
 
 [flake8]
 max-line-length: 88
+
+[pytest]
+asyncio_mode=auto
```

Comment:
what's this for?

Reply:
There is a `DeprecationWarning`  [1](https://github.com/pytest-dev/pytest-asyncio/blob/34436cd6653b1dd8e1836a99ca1412b850167f52/pytest_asyncio/plugin.py#L61).
```bash
 DeprecationWarning: The 'asyncio_mode' default value will change to 'strict' in future, please explicitly use 'asyncio_mode=strict' or 'asyncio_mode=auto' in pytest configuration file.
    config.issue_config_time_warning(LEGACY_MODE, stacklevel=2)
```
I know it shouldn't be part of this PR, but it's not a big change, so I did it.